include/swift/AST/DiagnosticsSema.def

```diff
@@ -2723,13 +2723,22 @@ WARNING(differentiable_nondiff_type_implicit_noderivative_fixit,none,
       "stored property %0 has no derivative because %1 does not conform to "
       "'Differentiable'; add an explicit '@noDerivative' attribute"
       "%select{|, or conform %2 to 'AdditiveArithmetic'}3",
-      (Identifier, Type, Identifier, bool))
-WARNING(differentiable_let_property_implicit_noderivative_fixit,none,
+      (/*propName*/ Identifier, /*propType*/ Type, /*nominalName*/ Identifier,
+       /*nominalCanDeriveAdditiveArithmetic*/ bool))
+WARNING(differentiable_immutable_wrapper_implicit_noderivative_fixit,none,
       "synthesis of the 'Differentiable.move(along:)' requirement for %1 "
-      "requires all stored properties to be mutable; use 'var' instead, or add "
-      "an explicit '@noDerivative' attribute"
+      "requires all stored properties not marked with `@noDerivative` to be "
+      "mutable; add '%0.wrappedValue.set', or add an explicit '@noDerivative' "
```

Comment:
* `.set` is not a user-facing identifier today. I'd suggest describing this in plain English, e.g. "a setter for 'Foo.wrappedValue'".
* Suggesting that users add a setter isn't always helpful. Users are most likely using a property wrapper that they did not define, and adding a setter isn't something that can be done retroactively. I'd suggest removing this suggestion.

Reply:
> * I'd suggest removing this suggestion.

Done in https://github.com/apple/swift/pull/31173/commits/f4b389c30b8726977b1ba325cdded9bf0a1c2572.

Alternatively, we could conditionally suggest "adding a setter for 'Wrapper.wrappedValue'" iff the wrapper type is defined in the same file or module.